multigeiger/multigeiger.ino

```diff
@@ -754,25 +751,21 @@ void DisplayStartscreen(void) {
 
   u8x8.clear();
```

Comment:
the above line clears the display.

Reply:
Isn't the library expected to be generic for all supported displays?

So, if it doesn't clear, file a bug against the library?

Reply:
And does it still happen with the latest lib release?